src/Sylius/Bundle/ApiBundle/Serializer/ProductVariantNormalizer.php

```diff
@@ -41,16 +44,20 @@ final class ProductVariantNormalizer implements ContextAwareNormalizerInterface,
     /** @var SectionProviderInterface */
     private $uriBasedSectionContext;
 
+    private IriConverterInterface $iriConverter;
+
     public function __construct(
         ProductVariantPricesCalculatorInterface $priceCalculator,
         ChannelContextInterface $channelContext,
         AvailabilityCheckerInterface $availabilityChecker,
-        SectionProviderInterface $uriBasedSectionContext
+        SectionProviderInterface $uriBasedSectionContext,
+        IriConverterInterface $iriConverter
     ) {
         $this->priceCalculator = $priceCalculator;
         $this->channelContext = $channelContext;
         $this->availabilityChecker = $availabilityChecker;
         $this->uriBasedSectionContext = $uriBasedSectionContext;
+        $this->iriConverter = $iriConverter;
```

Comment:
you don't use this service anywhere.

Reply:
I wanted to use it but accidentally I didn't. Fixed